cloud/aws-functions/orphaned_cloudformation.py

```diff
@@ -0,0 +1,61 @@
+# Remove unused cloudformations
+import logging
+import datetime
+import boto3
+import sys
+from time import sleep
+from botocore.exceptions import ClientError
+from boto3.exceptions import Boto3Error
+
+
+def isStackToTerminate(stack):
```

Comment:
is there some reason to use camel case here and snake case for other functions naming?